packages/block-editor/src/components/media-placeholder/styles.native.scss

```diff
@@ -60,13 +60,13 @@
 
 .addMediaButton {
 	color: $white;
-	background-color: $dark-gray-500;
+	background-color: #7b9ab1;
```

Comment:
I see this hex code is present in `_colors.scss` as `$toolbar-button`, we should replace it with the variable in all the places it's being used.

Reply:
Nice find! I didn't realize we already had this as a variable. I've updated to use the `$toolbar-button` variable (except in `packages/components/src/button/index.native.js` which uses `StyleSheet.create`, where I left a comment instead).

I was tempted to change a few other hard-coded color values to their respective variable names, but quickly realized there are a lot, and such a large change is probably deserving of its own PR. I can open a tech debt issue if one does not already exist.

Reply:
I opened an issue to track this here: https://github.com/wordpress-mobile/gutenberg-mobile/issues/1783